maas-api.yaml

```diff
@@ -1,555 +1,677 @@
-                realTime:
-                  $ref: '#/definitions/RealtimeData'
-            example:
-                - id: 803
-                  name: Metro Green Line  (803)
-                  mode: pt_pub_tram
-                  numberOfServices: 485
-                  stops:
-                    - 80314
-                    - 80312
-                    - 80307
-                    - 80305
-                    - 80301
-                - id: 806
-                  name: Metro Expo Line  (806)
-                  mode: pt_pub_tram
-                  numberOfServices: 562
-                  stops:
-                    - 80137
-                    - 80122
-                    - 80139
-                    - 80136
-                    - 80125
-                    - 80138
-                  routeColor:
-                    - blue: 198
-                    - green: 103
-                    - red: 0
-                  realtime:
-                    - alerts: false
-                    - positions: true
-                    - updates: false
-                - id: 804
-                  name: Metro Gold Line  (804)
-                  mode: pt_pub_tram
-                  numberOfServices: 120
-                  stops:
-                    - 80404
-                    - 80402
-                    - 80418
-                    - 80426
-                    - 80419
-                    - 80410
-                    - 80427
-                    - 80411
+                    description: Route name.
+                  description: Route names to retrieve information from. In case of any missmatch, it'll return error response. Use either this filter or `routeIDs`.
+                onlyRealTime:
+                  type: boolean
+                  description: Boolean to filter only routes with Real Time support. When true, returned route list will contain only routes with Real Time service.
+                  default: false
+                full:
+                  type: boolean
+                  description: Boolean if result should include full information for route/s or just a list with all their IDs.
+                  default: false
+              required:
+                - region
+                - agencyID
+              example:
+                regionID: US_CA_LosAngeles
+                agencyID: LACMTA_Rail
+                modes:
+                  - pt_pub_tram
+                full: true
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RoutesResponse'
         400:
           description: Bad request, most likely due to issues with the input or non-posible scenarios.
-          schema:
-            $ref: '#/definitions/InputError'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InputError'
 
-  /info/service.json:
+  /services/filter:
```

Comment:
can we just make this `/services`?

Reply:
Yep, good idea. Also same response schema here for `GET` and `POST` that way.